.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
-lib/**/*
+lib/**/*
+.secrets
```

Comment:
There should be a newline at the end of the file to adhere to standard conventions.